sections_flask-jwt-extended/app.py

```diff
@@ -0,0 +1,107 @@
+from flask import Flask, jsonify
+from flask_restful import Api
+from flask_jwt_extended import JWTManager
+
+from db import db
+from resources.user import UserRegister, UserLogin, TokenRefresh
+from resources.item import Item, ItemList
+from resources.store import Store, StoreList
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///data.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+api = Api(app)
+db.init_app(app)
+
+"""
+JWT related configurations began. The following functions includes:
+1) add claims to each jwt
+2) customize the token expired error message 
+"""
+app.config['JWT_SECRET_KEY'] = 'jose'   # we can also use app.secret like before, Flask-JWT-Extended can recognize both
+app.config['JWT_BLACKLIST_ENABLED'] = True  # enable blacklist feature
+app.config['JWT_BLACKLIST_TOKEN_CHECKS'] = ['access', 'refresh']  # allow blacklisting for access and refresh tokens
```

Comment:
Do we still need `refresh` here?

Reply:
It's true that we didn't blacklist any refresh token in the code. But I think we can keep it and talk about it, just to let the students know we can also blacklist a refresh token as well. In this way, we can freeze an account by all means.